src/os_maild/os_maild_client.c

```diff
@@ -135,7 +135,7 @@ MailMsg *OS_RecvMailQ(file_queue *fileq, struct tm *p,
     }
 
     /* We have two subject options - full and normal */
-    if (Mail->subject_full) {
+    if (Mail->subject_full == 0) {
```

Comment:
Can you elaborate?

Reply:
I just packaged OSSEC 3.0.0 and upgraded my monitor;

on 2.9.3, `maild.full_subject=1` is honored and I receive `OSSEC Alerts ...` (full) mails.
on 3.0.0, `maild.full_subject=1` is not honored and I receive `OSSEC Notifications ...` mails.
on 3.0.0, `maild.full_subject=0` is somewhat honored and I receive `OSSEC Alerts ...` (full) mails.

By the documentation, I did not expected the two last behaviour, but rather the contrary

I suspect this line should be `Mail->subject_full == 1` ?

Reply:
Since the behavior expected in the original issue didn't work if `Mail->subject_full == 1`, I'm guessing the answer is no. I think we should just remove this nonsense and not ever have to worry about it again.